plugins/inputs/win_wmi/win_wmi_test.go

```diff
@@ -75,3 +78,42 @@ func TestQuery(t *testing.T) {
 	// FreeSpace property was collected as a field
 	require.NotEmpty(t, acc.Metrics[0].Fields["FreeSpace"])
 }
+
+func TestMethod(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration test in short mode")
+	}
```

Comment:
This test will never run in automation/CI. You skip it during short, so `make test` will never run it. And since it doesn't contain Integration in the name `make integration` will never run it.

Reply:
Yeah I run it on the local box. Do you want me to make this an integration test?

Reply:
Yes please I would like to know we continue to test this.